src/pep8ext_naming.py

```diff
@@ -291,6 +291,25 @@ class ClassNameCheck(BaseASTCheck):
     Classes for internal use have a leading underscore in addition.
     """
     N801 = "class name '{name}' should use CapWords convention"
+    N818 = "class name '{name}' should use Error suffix as it is an Exception"
+
+    def get_class_def(self, name, parents):
+        for parent in parents:
+            for definition in parent.body:
+                class_definition = isinstance(definition, ast.ClassDef)
+                if class_definition and definition.name == name:
```

Comment:
nit: I think `is_class_definition` would be a more consistent name for this variable based on how we name similar things in this module.